plugins/experimental/server_push_preload/server_push_preload.cc

```diff
@@ -0,0 +1,109 @@
+#include <iostream>
+#include <regex>
+#include <sstream>
+#include <vector>
+#include <ts/ts.h>
+#include <ts/experimental.h>
+#include <atscppapi/GlobalPlugin.h>
+#include <atscppapi/utils.h>
+
+#define PLUGIN_NAME "server_push_preload"
+
+using namespace std;
+using namespace atscppapi;
+
+static regex linkRegexp("<([^>]+)>;.*rel=preload");
```

Comment:
This will match "xrel=preloadx". At least the attribute name should be checked strictly because attribute name is extensible (Non-standard unknown names have to be expected).